src/StringParser/Php.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Noodlehaus\StringParser;
+
+use Exception;
+use Noodlehaus\Exception\ParseException;
+use Noodlehaus\Exception\UnsupportedFormatException;
+
+/**
+ * PHP string parser
+ *
+ * @package    Config
+ * @author     Jesus A. Domingo <jesus.domingo@gmail.com>
```

Comment:
Please, add yourself as author of this class, removing the others.

Reply:
I've fixed this.